test/SPI/Inputs/spi_helper.swift

```diff
@@ -1,5 +1,8 @@
 /// Library defining SPI decls
 
+import Swift
+@_spi @_spi(ARandomSPI) import Swift
```

Comment:
What are you testing with these two `import Swift` lines? There’s most likely an implicit `import Swift` at work here too, so I’m wondering if it’s a good idea to use `Swift` in this test.

Reply:
This was meant to test the indirect importation of the SPI attribute in the binary and textual interfaces. If it can be problematic I'll remove it, I plan on adding more complex tests cases explicitly testing setup involving 3 modules.